### DISCUSSION TITLE
Can you restrict who can assign people to an issue?

### DISCUSSION THREAD

**Discussion by Lavean** [community/community#23210] [Projects and Issues]
<p>Hey community,</p>
<p>the title already says it all.<br>
In our repository, I would like to restrict who is allowed to assign someone to an issue.<br>
The problem is that we have some “trigger happy” developers haha who like to assign as much as possible to themselves, but that blocks the issue for someone else who might be more suitable or has an open time slot. It’s also not great for overview.</p>
<p>I didn’t find anything in the options. Is this possible at all?<br>
Thanks for your help!</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>:wave: Welcome!</p>
<p>I’m afraid this might have to be solved by a creative workplace policy for now. At present, <em>anyone with write permissions</em> can assign an issue, including to themselves.</p>
